### ISSUE TITLE
why enforce lowercase?

### ISSUE THREAD

**Issue by giovp** [broadinstitute/Tangram#71]
https://github.com/broadinstitute/Tangram/blob/3c87a25d49ad50b7f85ba68c67d31b46e38c3847/tangram/mapping_utils.py#L40-L41

think it'd be useful if this operation is optional.

**Comment by Hejin0701** [comment]
Hi @giovp ,

Thanks for your suggestion! The initial purpose of these two lines of commands are to get rid of the inconsistent capitalization of gene name. You are right, it is not always needed and thus it is better to make it optional. 

I have just updated Tangram function **pp_adatas** with an additional _parameter gene_to_lowercase_. The default value is set as _True_. If not needed, people can alway set it as _False_ to prevent conversion of the genes names to lower cases. One additional line of instructions is added before the **pp_adatas** function in tutorial.

**Comment by giovp** [comment]
thank you!

**Comment by yinan-wan0** [comment]
Hi,

It looks like this enforcing lower case was not only in pp_adatas, but also in tg.project_genes. After running it my sc_adata genes become all lower case. Can you please also provide an option to disable it there?

Thanks a lot!

**Comment by giovp** [comment]
Indeed! that's really problematic and give rise to all sorts of issues, am reopening this cause the issue is not resolved
